okapi-core/src/main/raml/okapi.raml

```diff
@@ -46,6 +46,66 @@ traits:
         enum: ["true","false","only"]
         required: false
         default: "true"
+  installParameters:
+    queryParameters:
+      async:
+        description: Whether to install in the background
+        type: boolean
+        required: false
+        default: false
+      deploy:
+        description: Whether to deploy (or undeploy if disabling)
+        type: boolean
+        required: false
+        default: false
+      ignoreErrors:
+        description: |
+          Okapi 4.2.0 and later, it is possible to ignore errors during the
+          install operation. This is done by supplying parameter `ignoreErrors=true`.
+          In this case, Okapi will try to upgrade all
+          modules in the modules list, regardless if one of them fails. However,
+          for individual modules, if they fail, their upgrade will not be commited.
+          This is an experimental parameter which was added to be able to inspect
+          all problem(s) with module upgrade(s).
+        type: boolean
+        required: false
+        default: false
+      invoke:
+        description: |
+         Whether to invoke for tenant init/permissions/purge. Use
+          "true" to invoke, "false" to not invoke. Any other value
+          is a regular expression that is matched against the module ID.
+          If that pattern matches, invoke is performed (same as "true").
+        type: string
+        required: false
+        default: "true"
+      parallel:
+        description: |
+          Number of parallel calls to tenant interfaces of modules.
+          A value of 1 means calls are happening in sequence.
+        type: integer
+        required: false
+        default: 1
+        minimum: 1
+      purge:
+        description: Disabled modules will also be purged.
+        type: boolean
+        required: false
+        default: false
+      reinstall:
+        description: Whether to install modules even if up-to-update
+        type: boolean
+        required: false
+        default: false
+      simulate:
+        description: Whether the installation is simulated
+        type: boolean
+        required: false
+        default: false
+      tenantParameters:
+        description: Parameters for Tenant init
```

Comment:
"Tenant" probably doesn't need to be capitalized. It isn't in other properties here.

Reply:
sure